sdk/identity/identity/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@azure/identity",
   "sdk-type": "client",
-  "version": "1.1.0-preview1",
+  "version": "1.1.0-preview1.0",
```

Comment:
is this correct? @danieljurek

Reply:
The original version should have been preview.1 and so this one should be preview.2